scripts/npmPublish.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Publishes the current version
+
+set -e # exit on errors
+
+BASE_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+echoe() {
+  # find all colors here: https://stackoverflow.com/questions/5947742/how-to-change-the-output-color-of-echo-in-linux#5947802
+  # Reset
+  Color_Off='\033[0m'       # Text Reset
+
+  # Bold High Intensity
+  BIBlack='\033[1;90m'      # Black
+  
+  # High Intensity backgrounds
+  On_IYellow='\033[0;103m'  # Yellow
+
+  echo ""
+  echo ""
+  echo -e "${On_IYellow}${BIBlack}${1}${Color_Off}"
+  echo ""
+}
+
+function setNpmToken {
+  if [ "$TRAVIS" = "true" ]; then
+    # setup token when running in travis
+    echo "setNpmToken"
+    echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc
+    npm whoami
+  fi
+}
+
+function checkGithubRelease {
+  cd $BASE_DIR/../client
+  NAME=$(node -p "require('./package.json').name")
+  CLIENT_VERSION=$(node -p "require('./package.json').version")
+  cd $BASE_DIR/../core/public
+  CORE_VERSION=$(node -p "require('./package.json').version")
+
+  if [ "CORE_VERSION" != "CLIENT_VERSION" ]; then
```

Comment:
Corrrect me if I´m wrong, but here we are comparing strings instead of the values from the variables